README.md

```diff
@@ -68,8 +68,8 @@ Mastodon acts as an OAuth2 provider so 3rd party apps can use the REST and Strea
 **Requirements:**
 
 - **PostgreSQL** 9.5+
-- **Redis**
-- **Ruby** 2.4+
+- **Redis** 4+
```

Comment:
This is probably part of another PR? Until Sidekiq 6.0 is merged I believe we are still compatible with Redis 3, for example.